modules/setting/queue.go

```diff
@@ -22,12 +20,8 @@ type QueueSettings struct {
 	BatchLength      int
 	ConnectionString string
 	Type             string
-	Network          string
```

Comment:
do we have reverences in example.ini or ohter docs left?

Reply:
Nope.